webapp/src/main/webapp/src/app/pages/importers/service-refs.dialog.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
We're missing the LICENSE header here. Could you please add it back?

Reply:
updated